design/template.md

```diff
@@ -0,0 +1,232 @@
+This template is adapted from Kubernetes Enchancements KEP template https://raw.githubusercontent.com/kubernetes/enhancements/a86942e8ba802d0035ec7d4a9c992f03bca7dce9/keps/NNNN-kep-template/README.md
```

Comment:
suggestion: By having this be actual text I think there's a decent chance that people won't remove it and it'll get in the way of the actual design doc.

I think it's nice to have this link but it could be a comment; failing that maybe we could move it right to the bottom of the template?

```suggestion
<!--
This template is adapted from Kubernetes Enchancements KEP template https://raw.githubusercontent.com/kubernetes/enhancements/a86942e8ba802d0035ec7d4a9c992f03bca7dce9/keps/NNNN-kep-template/README.md
-->
```

Reply:
Comment make sense yeah 👍🏼